src/main/twirl/com/lightbend/lagom/docs/sideNav.scala.html

```diff
@@ -22,7 +22,7 @@
     <li><a href="@context.path/community-process.html">Community process</a></li>
     <li><a href="@context.path/contributing.html">Contributor guidelines</a></li>
 </ul>
-<h3>Programs</h3>
+<!-- <h3>Programs</h3>
```

Comment:
Maybe it's a good idea to explain why this is commented out, for the benefit of future team members and our future selves 😄

Reply:
Yup, good idea, will do.

Reply:
Done.